lib/adroller/version.rb

```diff
@@ -1,3 +1,3 @@
 module Adroller
-  VERSION = '1.2.2'
```

Comment:
The version for this would have to be 2.0.0, based on Semantic Versioning. This change won't work on older uses of the gem since we are no longer relying on environment variables.

"Given a version number MAJOR.MINOR.PATCH, increment the:
    MAJOR version when you make incompatible API changes,
    MINOR version when you add functionality in a backwards-compatible manner, and
    PATCH version when you make backwards-compatible bug fixes."